ExternalScripts(DONOTINCLUDEINRELEASE)/scraper/quest/quest_formatter.py

```diff
@@ -35,7 +35,7 @@ def __load_json_file(self, file_name: str):
         return filtered_sorted_data
 
     def __sort_and_filter_data(self, data):
-        sorted_data = sorted(data, key=lambda x: x.get('questId', 0))
+        sorted_data = sorted(data, key=lambda x: int(x.get('questId', 0)))
```

Comment:
This changes the sorting so it sorts numbers as numbers like `1,2,3,10,20,100,200` and not as strings `1, 10, 100, 2, 20, 200, 3, ...`

Reply:
Perfect 👍🏻